Packages/Env/Sources/Env/TranslationType.swift

```diff
@@ -0,0 +1,18 @@
+import SwiftUI
+
+public enum TranslationType: String, CaseIterable {
+  case useServerIfPossible
+  case useDeepl
+  case useApple
+
+  public var description: LocalizedStringKey {
+    switch self {
+    case .useServerIfPossible:
+      "enum.translation-type.use-server-if-possible"
```

Comment:
Same for this, if we don't have it in localization dic, use the English string directly.